portality/templates/publisher/metadata.html

```diff
@@ -127,13 +127,22 @@ <h4 class="red">There is a problem with the submitted form.</h4>
     
     function removeAuthor() {
         event.preventDefault();
-        
+
+        var all_e = $('[id^=authors-][id$="container"]');
+        if (all_e.length === 1) {
+            console.log($('#remove_warning') == 0)
+            if ($('#remove_warning').length == 0) {
+                all_e.last().append('<div id="remove_warning"><ul class="red"><li>Please provide at least one author</li></ul><div>')
```

Comment:
Doesn't the validator already do this? If so, would it be better to only display the delete button if there's more than one author field?

Reply:
It does after clicking the submit button. But yes, that's actually a much better idea. Instead of displaying the warning it is better to just hide the X button when there is only one author. I'll change it and re-request the review when ready.